xbmc/cores/VideoPlayer/VideoRenderers/FrameBufferObject.cpp

```diff
@@ -53,10 +53,14 @@ CFrameBufferObject::CFrameBufferObject()
 
 bool CFrameBufferObject::IsSupported()
 {
+#if HAS_GLES >= 2
```

Comment:
unfortunately there is no `GL_EXT_framebuffer_object` in GLES so I'm not sure what to do.